MapboxNavigation/Geometry.swift

```diff
@@ -70,37 +69,37 @@ func -(left: RadianCoordinate2D, right: RadianCoordinate2D) -> RadianDistance {
 /*
  Returns the Haversine distance between two coordinates measured in degrees.
  */
-func -(left: CLLocationCoordinate2D, right: CLLocationCoordinate2D) -> CLLocationDistance {
+public func -(left: CLLocationCoordinate2D, right: CLLocationCoordinate2D) -> CLLocationDistance {
```

Comment:
Custom operators don’t exist in Objective-C. And this custom operator was a pretty bad idea in the first place.